pom.xml

```diff
@@ -94,7 +94,32 @@
 
     <modules>
         <module>hugegraph-server</module>
-        <module>hugegraph-pd</module>
-        <module>hugegraph-store</module>
     </modules>
+    <profiles>
+        <profile>
+            <id>stage</id>
+            <repositories>
+                <repository>
+                    <id>staged-releases</id>
+                    <url>https://repository.apache.org/content/groups/staging/</url>
+                </repository>
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>staged-releases</id>
+                    <url>https://repository.apache.org/content/groups/staging/</url>
+                </pluginRepository>
+            </pluginRepositories>
+        </profile>
+        <profile>
+            <id>exclude-jdk11-modules</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <modules>
+                <module>hugegraph-store</module>
+                <module>hugegraph-pd</module>
+            </modules>
+        </profile>
```

Comment:
Due to the current support for JDK 11 compilation in the hugegraph-store/pd modules, it is necessary to exclude these two modules in the JDK 8.